src/network/jsonwebtask.cpp

```diff
@@ -274,10 +274,10 @@ bool JsonWebTask::doStart(
             Qt::UniqueConnection);
 
     connect(m_pendingNetworkReply,
-#if QT_VERSION >= QT_VERSION_CHECK(5, 7, 0)
-            qOverload<QNetworkReply::NetworkError>(&QNetworkReply::error),
+#if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)
+            &QNetworkReply::errorOccurred,
 #else
-            QOverload<QNetworkReply::NetworkError>::of(&QNetworkReply::error),
+            qOverload<QNetworkReply::NetworkError>::of(&QNetworkReply::error),
```

Comment:
Typo: Capital Q

Reply:
Ouch, I wonder why it compiled locally...

Reply:
Because I already have Qt 5.15... Nevermind